Cargo.toml

```diff
@@ -62,6 +62,9 @@ zstd-compression = ["zstd"]
 zlib-compression = ["flate2"]
 snappy-compression = ["snap"]
 
+# DO NOT USE; see https://jira.mongodb.org/browse/RUST-569 for the status of FLE support in the Rust driver.
+fle = ["mongocrypt", "which"]
```

Comment:
Are we now referring to all of FLE as "Queryable Encryption"? Or is that just FLE 2? If not, should we call this `csfle`? Or have separate features for fle 1 and fle 2?

Reply:
"Queryable Encryption" is just FLE 2.  I don't think it's worth having distinct features; there's no cost to having FLE2 present if it's not used except for extra compilation, and even that's going to be nearly trivial.

I've renamed this to "csfle" because that seems to be what's consistently used for the overall feature set in public materials.